source/frontend/src/features/properties/filter/CoordinateSearch/CoordinateSearchForm.tsx

```diff
@@ -1,5 +1,6 @@
-import { Col, Container, Form } from 'react-bootstrap';
+import { Col, Row } from 'react-bootstrap';
 import styled from 'styled-components';
+import { clsx } from 'yet-another-react-lightbox';
```

Comment:
I'm not sure this is the proper import. I believe we use `import clsx from 'classnames';`